<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingResultWidget</class>
 <widget class="QWidget" name="TrainingResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorithms Training</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>690</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{
border-width: 2;
border-style: solid;
border-color: rgb(255, 170, 0);

}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="cbAB">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ada Boost</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbBAG">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B AG</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbDT">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decision Tree</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbDTW">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DTW</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbFSM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Finite State Machine</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbGMM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>GMM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHMMC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HMM Continuous</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHMMD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HMM Discrete</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbKNN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>KNN</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MinDist</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbRF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Forest</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Softmax</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSVM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SVM</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbANBC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ANBC</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbAB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbANBC">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbBAG">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbDT">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbDTW">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbFSM">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbGMM">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbHMMC">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbHMMD">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbKNN">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbRF">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbSM">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbMD">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbSVM">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="tAB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accAB">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tANBC">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accANBC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tBAG">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accBAG">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tDT">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="tFSM">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accFSM">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tDTW">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accDTW">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="accDT">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tGMM">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="tHMMD">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accHMMD">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tHMMC">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accHMMC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="accGMM">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tKNN">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="tRF">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accRF">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tMD">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accMD">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="accKNN">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tSVM">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accSVM">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tSM">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>9s</string>
    </property>
   </widget>
   <widget class="QLabel" name="accSM">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt  &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uDTW">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uHMMC">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uGMM">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uFSM">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uHMMD">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uKNN">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uRF">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uMD">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uAB">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uANBC">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uDT">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uBAG">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uSVM">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uSM">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btSelectAll">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btStartTesting">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Start testing !</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Algorithms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Accuracy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTestFileName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test file: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btTrainAgain">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Train again &gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
